scripts/run-examples.sh

```diff
@@ -6,6 +6,8 @@
 pkill hss-server
 ./out/bin/hss-server -network_type sctp >/dev/null 2>&1 &
 
+sleep 2
```

Comment:
Please tell me why you included this

Reply:
@takehaya 
It is because run the test is faster than initializing the hss server.


Test execution sometimes fails.

https://github.com/bbsakura/xk6-diameter/actions/runs/13128052232/job/36628073507#step:7:12